perfkitbenchmarker/benchmarks/mysql_service_benchmark.py

```diff
@@ -0,0 +1,773 @@
+# Copyright 2014 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""MySQL Service Benchmarks.
+
+This is a set of benchmarks that measures performance of MySQL Databases on
+managed MySQL services.
+
+- On AWS, we will use RDS+MySQL.
+- On GCP, we will use Cloud SQL v2 (Performance Edition). As of July 2015, you
+ will need to request to whitelist your GCP project to get access to cloud sql
+ v2. Follow instructions on your GCP's project console to do that.
+- On Azure, we will use the MySQL service whenever it becomes availble.
```

Comment:
Actually PKB already has a benchmark that will create a VM, install mySQL on it, and run sysbench OLTP load. It's called sysbench_oltp_benchmarks and it supports all 3 providers. 

For this one, it's specifically for the managed mysql *service*. And good suggestion on changing the comments there to include other providers.

Done. Updated the comments per suggestion.